README.md

```diff
@@ -49,23 +51,59 @@ export EVENTLY_TOKEN="your-token-here"
 * [`evently registry:list-events`](#evently-registrylist-events)
 * [`evently registry:new`](#evently-registrynew)
 
+## `evently append:atomic`
+
+Appends a serial event to the ledger
+
+```
+USAGE
+  $ evently append:atomic [-t <value>] [-f <value>] [-b <value>]
+
+FLAGS
+  -b, --body=<value>   Read data from argument. If neither --body nor --file is given, STDIN is used.
+  -f, --file=<value>   Read event data from this file. If neither --body nor --file is given, STDIN is used.
+  -t, --token=<value>  [default: NOT-SET] Access token for your ledger.
+
+DESCRIPTION
+  Appends a serial event to the ledger
+
+EXAMPLES
+  $ evently append:atomic <<EVENT
+  {
+    entity: "thermostat",
+    event: "temperature-recorded",
+    key: "thermostat1",
+    meta: {causation: "14rew3494"},
+    data:{celsius: 18.5},
+    selector: {
+      selectorId: "foo",
+      mark: "bar"
+    }
```

Comment:
I've added these to the JSON body instead of using a separate argument. I can go either way with this, but it felt weird to have everything in the JSON body except 'selector'.

Reply:
If we keep this, I could also refactor all of this to instead make the event variant (atomic, factual, serial) a positional option so we don't have 3 mostly-identical classes.